cicecore/cicedyn/infrastructure/io/io_pio2/ice_history_write.F90

```diff
@@ -404,15 +405,14 @@ subroutine ice_write_hist (ns)
 
       ! bounds fields are required for CF compliance
       ! dimensions (nx,ny,nverts)
-      ! bounds inherit attributes
-      var_nverts(n_lont_bnds) = coord_attributes('lont_bounds','und','und','und')
-      var_nverts(n_latt_bnds) = coord_attributes('latt_bounds','und','und','und')
-      var_nverts(n_lonu_bnds) = coord_attributes('lonu_bounds','und','und','und')
-      var_nverts(n_latu_bnds) = coord_attributes('latu_bounds','und','und','und')
-      var_nverts(n_lonn_bnds) = coord_attributes('lonn_bounds','und','und','und')
-      var_nverts(n_latn_bnds) = coord_attributes('latn_bounds','und','und','und')
-      var_nverts(n_lone_bnds) = coord_attributes('lone_bounds','und','und','und')
-      var_nverts(n_late_bnds) = coord_attributes('late_bounds','und','und','und')
+      var_nverts(n_lont_bnds) = coord_attributes('lont_bounds','longitude bounds (T-cell)','degrees_east','und')
+      var_nverts(n_latt_bnds) = coord_attributes('latt_bounds','latitude bounds (T-cell)','degrees_north','und')
+      var_nverts(n_lonu_bnds) = coord_attributes('lonu_bounds','longitude bounds (U-cell)','degrees_east','und')
+      var_nverts(n_latu_bnds) = coord_attributes('latu_bounds','latitude bounds (U-cell)','degrees_north','und')
+      var_nverts(n_lonn_bnds) = coord_attributes('lonn_bounds','longitude bounds (N-cell)','degrees_east','und')
+      var_nverts(n_latn_bnds) = coord_attributes('latn_bounds','latitude bounds (N-cell)','degrees_north','und')
+      var_nverts(n_lone_bnds) = coord_attributes('lone_bounds','longitude bounds (E-cell)','degrees_east','und')
```

Comment:
Do we use the 'N-cell' and 'E-cell' terminology anywhere else?  Is that specified by CF rules?

Reply:
Here are the comments from ice_grid.F90:

        lonn_bounds, & ! longitude of gridbox corners for N point
         latn_bounds, & ! latitude of gridbox corners for N point
         lone_bounds, & ! longitude of gridbox corners for E point
         late_bounds    ! latitude of gridbox corners for E point

Maybe I should say corners here.

Reply:
I think setting bounds attributes can cause problems with CF compliance.  I'm not entirely sure what the requirement is.  At a minimum, the attributes must match the variable they are associated with exactly.  If the attributes are not defined, they will be inherited from the associated variable.  I think I ran into some issues with the CF checkers with the bounds attributes.  We should definitely run the updated history files through the CF checkers before merging just so we know where we stand.